torchvision/prototype/transforms/functional/_color.py

```diff
@@ -1,152 +1,168 @@
-from typing import Union
-
-import PIL.Image
 import torch
 from torchvision.prototype import features
 from torchvision.transforms import functional_pil as _FP, functional_tensor as _FT
 
 
-# shortcut type
-DType = Union[torch.Tensor, PIL.Image.Image, features._Feature]
-
 adjust_brightness_image_tensor = _FT.adjust_brightness
 adjust_brightness_image_pil = _FP.adjust_brightness
 
 
-def adjust_brightness(inpt: DType, brightness_factor: float) -> DType:
-    if isinstance(inpt, features._Feature):
+def adjust_brightness(inpt: torch.Tensor, brightness_factor: float) -> torch.Tensor:
+    if isinstance(inpt, torch.Tensor) and (
+        torch.jit.is_scripting() or torch.jit.is_tracing() or not isinstance(inpt, features._Feature)
```

Comment:
JIT doesn't like it if we move the `torch.jit.is_scripting() or torch.jit.is_tracing()` from the if and putting it on a separate variable before the if...